docs/docs/airtable/oauth-migration-2024-02/README.md

```diff
@@ -0,0 +1,105 @@
+# Update to the Airtable Integration on Pipedream (January 2024)
+
+Effective February 1st 2024, Airtable's API Key authentication method will be deprecated. To learn more about this change, please visit Airtable’s [dedicated support page](https://support.airtable.com/docs/airtable-api-key-deprecation-notice).
+
+### How will this impact my workflows?
+
+Starting February 1st 2024, all Pipedream steps using the legacy Airtable (API Key) integration including triggers and actions will no longer be able to authenticate with Airtable. 
+
+### What do I need to do? 
+<br/> 
+
+1. **Reconnect your Airtable account**: 
+
+- Visit the [accounts page in Pipedream](https://pipedream.com/accounts)
+- Search for Airtable and connect your account
+- This newer Pipedream integration uses OAuth instead of an API Key
+
+![Airtable Account Connection](https://res.cloudinary.com/dpenc2lit/image/upload/v1704392183/Screenshot_2024-01-04_at_10.16.12_AM_le364k.png)
+
+You can determine which workflows are connected to the legacy Airtable (API Key) app by expanding the account row on your Airtable (API Key) account connection.
+![Airtable Accounts](https://res.cloudinary.com/dpenc2lit/image/upload/v1704347928/Screenshot_2024-01-03_at_9.58.43_PM_haaqlb.png)
+
+2. **Update Your Workflows**: 
+
+- After reconnecting to Airtable via OAuth, you'll need to update your existing workflows that use the legacy Airtable app.
+- Remove any legacy Airtable sources and re-add the source using the new Airtable app
+- Remove any legacy Airtable actions and re-add them using the new Airtable app
+
+<br/>
+
+3. **If you're using Airtable in code:**
+
+- Change any of your code steps to reference `airtable_oauth` instead of `airtable`. 
+- Modify your authorization headers accordingly
+
+In Node.js, you would modify your authorization header like this:
+
+    `"Authorization": ${this.airtable.$auth.api_key}`
+
+    to 
+
+    ```Authorization: `Bearer ${this.airtable_oauth.$auth.oauth_access_token}` ```
```

Comment:
you probably have to escape the backticks that wrap Bearer, like so:
```suggestion
    `Authorization: \`Bearer ${this.airtable_oauth.$auth.oauth_access_token}\``
```

does that work?

Reply:
This seemed to work:

``` Authorization: `Bearer ${this.airtable_oauth.$auth.oauth_access_token}` ```

Would that be acceptable? In preview, it looks good.

Reply:
this is what I see 
![Screenshot by Dropbox Capture](https://github.com/PipedreamHQ/pipedream/assets/20780753/770b42b4-f422-4e33-94ad-4aff5c9f3672)

Reply:
Sorry, I made some changes to lessen the indent and it worked. Was having issues committing my change, but now it should be working!